test/axes/timeAxisTests.ts

```diff
@@ -1,416 +1,386 @@
+      let scale = new Plottable.Scales.Time();
       assert.throws(() => new Plottable.Axes.Time(scale, "left"), "horizontal");
       assert.throws(() => new Plottable.Axes.Time(scale, "right"), "horizontal");
-  });
+    });
 
-  it("cannot change time axis orientation to vertical", () => {
+    it("cannot change to a vertical orientation", () => {
+      let scale = new Plottable.Scales.Time();
+      let originalOrientation = "bottom";
+      let axis = new Plottable.Axes.Time(scale, originalOrientation);
       assert.throws(() => axis.orientation("left"), "horizontal");
       assert.throws(() => axis.orientation("right"), "horizontal");
-      assert.strictEqual(axis.orientation(), "bottom", "orientation unchanged");
-  });
-
-  it("Computing the default ticks doesn't error out for edge cases", () => {
-    let svg = TestMethods.generateSVG(400, 100);
-    scale.range([0, 400]);
-
-    // very large time span
-    assert.doesNotThrow(() => scale.domain([new Date(0, 0, 1, 0, 0, 0, 0), new Date(50000, 0, 1, 0, 0, 0, 0)]));
-    axis.renderTo(svg);
-
-    // very small time span
-    assert.doesNotThrow(() => scale.domain([new Date(0, 0, 1, 0, 0, 0, 0), new Date(0, 0, 1, 0, 0, 0, 100)]));
-    axis.renderTo(svg);
-
-    svg.remove();
+      assert.strictEqual(axis.orientation(), originalOrientation, "orientation unchanged");
+      axis.destroy();
+    });
   });
 
-  it("Tick labels don't overlap", () => {
-    let svg = TestMethods.generateSVG(400, 100);
-    scale.range([0, 400]);
-
-    function checkDomain(domain: any[]) {
-      scale.domain(domain);
+  describe("rendering in edge case scenarios", () => {
+    it("does not error when setting the domain to a large span", () => {
+      let scale = new Plottable.Scales.Time();
+      let axis = new Plottable.Axes.Time(scale, "bottom");
+      let svg = TestMethods.generateSVG();
       axis.renderTo(svg);
 
-      function checkLabelsForContainer(container: d3.Selection<void>) {
-        let visibleTickLabels = container
-                .selectAll("." + Plottable.Axis.TICK_LABEL_CLASS)
-                .filter(function(d: any, i: number) {
-                  return d3.select(this).style("visibility") === "visible";
-                });
-        let numLabels = visibleTickLabels[0].length;
-        let clientRect1: ClientRect;
-        let clientRect2: ClientRect;
-        for (let i = 0; i < numLabels; i++) {
-          for (let j = i + 1; j < numLabels; j++) {
-            clientRect1 = (<Element> visibleTickLabels[0][i]).getBoundingClientRect();
-            clientRect2 = (<Element> visibleTickLabels[0][j]).getBoundingClientRect();
-
-            assert.isFalse(Plottable.Utils.DOM.clientRectsOverlap(clientRect1, clientRect2), "tick labels don't overlap");
-          }
-        }
-      }
+      // very large time span
+      assert.doesNotThrow(() => scale.domain([new Date(0, 0, 1, 0, 0, 0, 0), new Date(5000, 0, 1, 0, 0, 0, 0)]));
```

Comment:
Do we know what bug this test was intended to solve? It looks like it was added in #716, when the Time Axis was first added.